qiita_pet/templates/stats.html

```diff
@@ -49,26 +49,40 @@
 
 {% end %}
 {% block content %}
+  <div id="jumbotron" class="jumbotron">
+    <table width="100%">
+    <thead>
+      <tr>
+        <th>Studies</th>
+        <th>Samples</th>
+        <th>Users</th>
+      </tr>
+    </thead>
+      <tr>
+        <td>
+          {% for k in number_studies %}
+            <i>{{k}}</i>: {{number_studies[k]}}
```

Comment:
Can you put each of this values in one row? I think it looks better than one next to each other.